src/SUMMARY.md

```diff
@@ -35,6 +35,7 @@
       - [iwd](./config/network/iwd.md)
       - [NetworkManager](./config/network/networkmanager.md)
       - [ConnMan](./config/network/connman.md)
+   - [NFS](./config/nfs.md)
```

Comment:
Perhaps name the page "Network File System - NFS" ? Or just "Network File System".

Reply:
Just saw @flexibeast's comment. Use the title she suggested, then.